hoot-services/src/main/java/hoot/services/geo/BoundingBox.java

```diff
@@ -22,7 +22,7 @@
  * This will properly maintain the copyright information. DigitalGlobe
  * copyrights will be updated automatically.
  *
- * @copyright Copyright (C) 2015, 2016, 2018 DigitalGlobe (http://www.digitalglobe.com/)
+ * @copyright Copyright (C) 2015, 2016, 2018, 2019 DigitalGlobe (http://www.digitalglobe.com/)
```

Comment:
Don't know why but the copyright script updated this java file.  Maybe there is something wrong with `develop`, maybe something is wrong with the script.  Either way it is updated.